### DISCUSSION TITLE
Release Source Code files (zip) without Version (tag)

### DISCUSSION THREAD

**Discussion by outsourcetoasia** [community/community#54099] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hello Community, 
I am new to this so please forgive me if I choose the wrong place. My problem is like this:
I create in a repository (lets call it git-test) a release. Tag is 1.0.0 (for example)
Everything works smooth and I have also 2 Source code files (zip and tar)
My problem is just that INSIDE this files the folder is git-test-1.0.0 but I need just git-test.
I was looking for any settings to change that but didn't found anything.
Is it possible, if yes where can I set it?
The reason is, I like to use this zip file direct for WordPress plugin updates.
I say already thank you in advanced.
Tom

**Comment by airtower-luna** [comment, marked as answer]
> Is it possible, if yes where can I set it?

No, there's no option for that.

What you can do is create your own archive with the structure you want and add it to the release (in addition to the automatically provided ones), GitHub calls this "release assets". You could even automate that with an Actions workflow that runs for each release.

See: https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases

**Reply by outsourcetoasia** [reply]
Thank you so much for your fast reply. Awesome.
Even if I hoped for another outcome. A template in the settings would be so cool for WP developers because you could use the release asset "out of the box".
